src/gc/gc.cpp

```diff
@@ -11615,9 +11613,9 @@ void gc_heap::adjust_limit_clr (uint8_t* start, size_t limit_size, size_t size,
     }
 
     //this portion can be done after we release the lock
-    if (seg == ephemeral_heap_segment)
+    if (seg == ephemeral_heap_segment ||
+       (seg == nullptr && gen_number == 0 && limit_size >= CLR_SIZE / 2))
```

Comment:
> (seg == nullptr && gen_number == 0 && limit_size >= CLR_SIZE / 2)) [](start = 6, length = 67)

nit - the coding convention in GC is to always use () around each expression -

 if ((seg == ephemeral_heap_segment) ||
     ((seg == nullptr) && (gen_number == 0) && (limit_size >= CLR_SIZE / 2)))

Reply:
Done.